runtime/integration-tests/src/pools/utils/accounts.rs

```diff
@@ -127,25 +239,185 @@ impl std::str::FromStr for Keyring {
+/// * Keyring::TrancheInvestor(28)
+/// * Keyring::TrancheInvestor(29)
+/// * Keyring::TrancheInvestor(30)
+/// * Keyring::TrancheInvestor(31)
+/// * Keyring::TrancheInvestor(32)
+/// * Keyring::TrancheInvestor(33)
+/// * Keyring::TrancheInvestor(34)
+/// * Keyring::TrancheInvestor(35)
+/// * Keyring::TrancheInvestor(36)
+/// * Keyring::TrancheInvestor(37)
+/// * Keyring::TrancheInvestor(38)
+/// * Keyring::TrancheInvestor(39)
+/// * Keyring::TrancheInvestor(40)
+/// * Keyring::TrancheInvestor(41)
+/// * Keyring::TrancheInvestor(42)
+/// * Keyring::TrancheInvestor(43)
+/// * Keyring::TrancheInvestor(44)
+/// * Keyring::TrancheInvestor(45)
+/// * Keyring::TrancheInvestor(46)
+/// * Keyring::TrancheInvestor(47)
+/// * Keyring::TrancheInvestor(48)
+/// * Keyring::TrancheInvestor(49)
+/// * Keyring::TrancheInvestor(50)
+pub fn default_accounts() -> Vec<Keyring> {
+	let mut standard = vec![
+		Keyring::Admin,
+		Keyring::Alice,
+		Keyring::Bob,
+		Keyring::Ferdie,
+		Keyring::Charlie,
+		Keyring::Dave,
+		Keyring::Eve,
+	];
+	standard.extend(default_investors());
+	standard
+}
+
+/// Returns a Vector of default investor accounts
+///
+/// Accounts:
+/// * Keyring::TrancheInvestor(1)
+/// * Keyring::TrancheInvestor(2)
+/// * Keyring::TrancheInvestor(3)
+/// * Keyring::TrancheInvestor(4)
+/// * Keyring::TrancheInvestor(5)
+/// * Keyring::TrancheInvestor(6)
+/// * Keyring::TrancheInvestor(7)
+/// * Keyring::TrancheInvestor(8)
+/// * Keyring::TrancheInvestor(9)
+/// * Keyring::TrancheInvestor(10)
+/// * Keyring::TrancheInvestor(11)
+/// * Keyring::TrancheInvestor(12)
+/// * Keyring::TrancheInvestor(13)
+/// * Keyring::TrancheInvestor(14)
+/// * Keyring::TrancheInvestor(15)
+/// * Keyring::TrancheInvestor(16)
+/// * Keyring::TrancheInvestor(17)
+/// * Keyring::TrancheInvestor(18)
+/// * Keyring::TrancheInvestor(19)
+/// * Keyring::TrancheInvestor(20)
+/// * Keyring::TrancheInvestor(21)
+/// * Keyring::TrancheInvestor(22)
+/// * Keyring::TrancheInvestor(23)
+/// * Keyring::TrancheInvestor(24)
+/// * Keyring::TrancheInvestor(25)
+/// * Keyring::TrancheInvestor(26)
+/// * Keyring::TrancheInvestor(27)
+/// * Keyring::TrancheInvestor(28)
+/// * Keyring::TrancheInvestor(29)
+/// * Keyring::TrancheInvestor(30)
+/// * Keyring::TrancheInvestor(31)
+/// * Keyring::TrancheInvestor(32)
+/// * Keyring::TrancheInvestor(33)
+/// * Keyring::TrancheInvestor(34)
+/// * Keyring::TrancheInvestor(35)
+/// * Keyring::TrancheInvestor(36)
+/// * Keyring::TrancheInvestor(37)
+/// * Keyring::TrancheInvestor(38)
+/// * Keyring::TrancheInvestor(39)
+/// * Keyring::TrancheInvestor(40)
+/// * Keyring::TrancheInvestor(41)
+/// * Keyring::TrancheInvestor(42)
+/// * Keyring::TrancheInvestor(43)
+/// * Keyring::TrancheInvestor(44)
+/// * Keyring::TrancheInvestor(45)
+/// * Keyring::TrancheInvestor(46)
+/// * Keyring::TrancheInvestor(47)
+/// * Keyring::TrancheInvestor(48)
+/// * Keyring::TrancheInvestor(49)
+/// * Keyring::TrancheInvestor(50)
+pub fn default_investors() -> Vec<Keyring> {
```

Comment:
how about (0..50).iter().map(|idx| Keyring::TrancheInvestor(idx+1)).collect()

Reply:
(though actually, keeping Investor 0 might also be fine?)

Reply:
Nice, yes will do that in the follow up PR! And you are right, I will keep the 0 investor. Already changed that in the follow up, but wanna close this first.